README.md

```diff
@@ -108,6 +108,17 @@ such a server can be launched with `python -m smtpd -n -c DebuggingServer localh
 
 Alternatively only unit tests can be run by doing `cargo test --lib`.
 
+## Troubleshooting
+
+These are general steps to be followed when troubleshooting SMTP related issues.
+- Ensure basic connectivity, ensure requisite ports are open and daemons are listening.
```

Comment:
I think markdown prefers to have an empty line before and after a list, although some implementations seem more forgiving 

```suggestion
These are general steps to be followed when troubleshooting SMTP related issues.

- Ensure basic connectivity, ensure requisite ports are open and daemons are listening.
```